src/utils/translation-helpers.ts

```diff
@@ -20,5 +21,5 @@ export const getLanguageString = (labels?: LanguageString) => {
   if (Object.keys(labels).includes(preferredLanguageCode)) {
     return labels[preferredLanguageCode];
   }
-  return Object.values(labels)[0];
+  return isEmpty(Object.values(labels)[0]) ? '' : Object.values(labels)[0];
```

Comment:
Litt skeptisk til å ta inn nye pakker for å løse ting man kan løse relativt greit uten også. Vet aldri helt hva man får (hva som ligger i koden der) med mindre man følger med for alle commits, i tillegg til at man får en ekstra pakke man bør holde oppdatert.

Da foretrekker jeg egentlig heller å oppdatere if-en øverst i metoden til:
```
if (!labels || Object.keys(labels).length === 0) {
   return '';
}
```

Reply:
helt enig så klart :-)  Men jeg hadde kjørt meg fast og endte opp men en lang sjekk for å få det til å funke. Dermed tenkte jeg det var bedre men en enkel-løsning. Dessuten har jeg god erfaring meg "just"-bibliotekene. De er så minimalistiske som man får dem og uten videre avhengigheter. (https://github.com/angus-c/just)
Tok selvsagt å endret til ditt løsningsforslag :-)